style.css

```diff
@@ -0,0 +1,5 @@
+.head {
+  width: 50vw;
+  height: 30vh;
+  color: blue !important;
```

Comment:
- [ ] Its best to not use the `!important` tag as much as possible. The only thing that can override an !important tag is another !important tag. By using it one once, you potentially end up with a CSS file that is full of !important tags, which is not ideal. `If all your styles are !important, then none of your styles are important.`